src/V3Inst.cpp

```diff
@@ -475,6 +475,34 @@ class InstDeVisitor final : public VNVisitor {
         }
     }
 
+    void visit(AstArg* nodep) override {
```

Comment:
Should this instead be a visitor on the AstArraySel?  Test using arrays places other than in function arguments.

Reply:
That makes sense, implemented.